src/Symfony/Component/Config/Builder/ConfigBuilderGenerator.php

```diff
@@ -102,6 +120,29 @@ private function writeClasses(): void
         $this->classes = [];
     }
 
+    private function buildConfigureOption(ArrayNode $node, ClassBuilder $class): void
+    {
+        $class->addUse('JetBrains\PhpStorm\ArrayShape');
```

Comment:
```suggestion
        $class->addUse(JetBrains\PhpStorm\ArrayShape::class);
```

(this works even if the dependency isn't installed)